packages/rum/src/domain/rumEventsCollection/view/viewCollection.spec.ts

```diff
@@ -25,6 +25,7 @@ describe('viewCollection', () => {
     const { lifeCycle, rawRumEvents } = setupBuilder.build()
     const view = {
       cumulativeLayoutShift: 1,
+      customTimings: {},
```

Comment:
it could be nice to have a test with some values for this file

Reply:
I agree, I will add them.